.github/workflows/duplicate-external-pr.yml

```diff
@@ -0,0 +1,39 @@
+name: Duplicate External PR and Run E2E Tests
+on:
+    issue_comment:
+        types: [created]
+jobs:
+    duplicate-and-test:
+        if: github.event.issue.pull_request && contains(github.event.comment.body, '/duplicate-external-pr')
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check user for team affiliation
+              id: check-user-for-team-affiliation
+              uses: tspascoal/get-user-teams-membership@v3
+              with:
+                  username: ${{ github.event.comment.user.login }}
+                  organization: lightdash
+                  team: engineering
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            - name: Stop workflow if user is no member
+              if: steps.check-user-for-team-affiliation.outputs.isTeamMember == 'false'
+              run: |
+                  echo "You have no rights to trigger this job."
+                  exit 1
+            - name: Checkout code
+              uses: actions/checkout@v2
+              with:
+                  ref: ${{ github.event.issue.pull_request.head.ref }}
+            - name: Duplicate external PR
+              env:
+                  ISSUE_NUMBER: ${{ github.event.issue.number }}
+                  PR_TITLE: ${{ github.event.issue.title }}
+                  REF: ${{ github.event.issue.pull_request.head.ref }}
+                  DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
+                  ASSIGNEE: ${{ github.event.comment.user.login }}
+              run: |
+                  git config user.name "github-actions[bot]"
+                  git config user.email "github-actions[bot]@users.noreply.github.com"
+                  git checkout -b duplicate-${{ env.ISSUE_NUMBER }} ${{ env.REF }}
```

Comment:
Would be awesome to support the case where the branch already exists.
For the cases when we ask for changes, we could run the command to update the branch/PR.

Reply:
Great! I've added this check as well.

Checks if branch exists already - if so, update the branch


https://github.com/lightdash/lightdash/pull/8986/commits/0e55f10760709a5467c17b8a25d1e0a7ee686e8a